src/wallet/rpcwallet.cpp

```diff
@@ -2540,15 +2554,113 @@ static UniValue loadwallet(const JSONRPCRequest& request)
     return obj;
 }
 
-static UniValue createwallet(const JSONRPCRequest& request)
+static std::map<std::string,WalletFlags> walletflagmap{
+    {"avoidreuse", WALLET_FLAG_AVOIDREUSE},
+    {"disable_private_keys", WALLET_FLAG_DISABLE_PRIVATE_KEYS},
+};
+
+static UniValue getwalletflags(const JSONRPCRequest& request)
```

Comment:
I'd prefer not to add this new method. Can we just add a `wallet_flags` field to the `getwalletinfo` return object?

Reply:
Removing it. There's already a "private_keys_enabled" and "avoid_reuse" entry in the returned value, so I don't think a wallet_flags is even necessary.